README.md

```diff
@@ -95,3 +94,7 @@ root
 - [**@aragon/api**](https://github.com/aragon/aragon.js/tree/master/packages/aragon-api): Aragon client application API.
 - [**@aragon/ui**](https://github.com/aragon/aragon-ui): Aragon UI components (in React).
 - [**@aragon/buidler-aragon**](https://github.com/aragon/buidler-aragon): Aragon Buidler plugin.
+
+### Aditional Information
+
+We have an additional script `copy-artifacts`. We use this script to get the files of the pre-compiled artifacts from `@aragon/abis`. Their purpose is to use it to deploy the aragonOS framework during local development. In a similar way, they can be used during tests.
```

Comment:
```suggestion
There is an additional npm script called `copy-artifacts`. We use this script to load the the pre-compiled artifacts from `@aragon/abis` into your local environment to allow for easily deploying aragonOS during development. In a similar fashion, they are also useful for scaffolding the test environment.

You should never have to run this command yourself, but in case you've deleted the local `./artifacts` folder, you can run `npm run copy-artifacts` to re-create it.
```

Do we also have `artifacts/` in the default .gitignore?

Reply:
Yes, we have